UPGRADING_DEPENDENCIES.md

```diff
@@ -26,10 +26,12 @@ To add or update a new dependency, use the `go get` command:
 
 ```bash
 # Pick the latest tagged release.
-go get example.com/some/module/pkg
+# The GO111MODULE variable can be omitted when the code isn't located in GOPATH.
+GO111MODULE=on go get example.com/some/module/pkg
```

Comment:
Didn't realize that we have this information documented here as well. I have an issue #17652 for improving the contribution guideline so will add some information about it would be nice to link to this documentation from contribution guideline.

Reply:
I think we should strive towards not having this info in two places. 
If we really need it, one could link to the other.